src/app/pages/network/components/interface-form/interface-form.component.ts

```diff
@@ -217,6 +222,15 @@ export class InterfaceFormComponent implements OnInit {
         this.isLoading = false;
         this.core.emit({ name: 'NetworkInterfacesChanged', data: { commit: false, checkin: false }, sender: this });
         this.slideInService.close();
+
+        this.ws.call('interface.default_route_will_be_removed').pipe(untilDestroyed(this)).subscribe((res) => {
```

Comment:
filter(Boolean) to avoid “res” and nesting

Reply:
Thanks. Fixed it